PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutRequest.java

```diff
@@ -60,7 +62,6 @@ public class PayPalNativeCheckoutRequest extends PayPalNativeRequest implements
      *               for mismatches between this client-side amount and the final amount in the Transaction
      *               are determined by the gateway.
      **/
-    @Deprecated
```

Comment:
Same comment - we can leave this specific constructor deprecated since it was deprecated previously for a different reason

Reply:
addressed here bd7d29279c755b3029da1b6181835e43ef6e60bc